app/components/careers_video_card_component/careers_video_card_component.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="careers-video-card">
+  <%= render YouTubeEmbedComponent.new(video_url: @video_url) %>
+  <div class="careers-video-card--content">
```

Comment:
These will need updating to the double underscore version, see comment on SCSS file